modules/my_health/spec/request/v1/messaging_preferences_spec.rb

```diff
@@ -8,100 +8,148 @@
   include SM::ClientHelpers
   include SchemaMatchers
 
+  let(:mhv_account_type) { 'Premium' }
   let(:va_patient) { true }
   let(:current_user) { build(:user, :mhv, va_patient:, mhv_account_type:) }
 
   before do
-    allow(SM::Client).to receive(:new).and_return(authenticated_client)
     sign_in_as(current_user)
   end
 
-  context 'Basic User' do
-    let(:mhv_account_type) { 'Basic' }
+  context 'when sm session policy is enabled' do
+    before do
+      Flipper.enable(:mhv_sm_session_policy)
```

Comment:
same here, is you allow(Flipper)... you won't have the possibility of running into flaky specs